example2/index.html

```diff
@@ -0,0 +1,383 @@
+          <h2>Logout</h2>
+
+          <div id="logout">
+            <code>logout()</code>
+            <form class="form-inline" onsubmit="event.preventDefault(); logout()">
+              <button type="submit" class="btn btn-primary mb-2">Logout</button>
+            </form>
+
+            <textarea id="terminal" class="form-control" cols="20" rows="6" readonly></textarea>
+
+            <script>
+              function logout() {
+                return logP($('#logout #terminal')[0])(skygear.auth.logout())
+              }
+          </script>
+          </div>
+        </div>
+
+        <!-- User Discovery -->
+        <div class="section section-user-discovery">
+          <h2>User Discovery</h2>
+
+          <div id="whoami">
+            <code>whoami()</code>
+            <form class="form-inline" onsubmit="event.preventDefault(); whoami()">
+              <button type="submit" class="btn btn-primary mb-2">Who am I?</button>
+            </form>
+
+            <textarea id="terminal" class="form-control" cols="20" rows="6" readonly></textarea>
+
+            <script>
+              function whoami() {
+                return logP($('#whoami #terminal')[0])(skygear.auth.whoami())
+              }
+            </script>
+          </div>
+
+          <div id="discoverUserByEmails">
+            <code>discoverUserByEmails([email])</code>
+            <form class="form-inline" onsubmit="event.preventDefault(); discoverUserByEmails()">
+              <div class="form-group mb-2">
+                <label for="email" class="sr-only">Email</label>
+                <input type="email" class="form-control" id="email" placeholder="Email" required>
+              </div>
+              <button type="submit" class="btn btn-primary mb-2">Discover</button>
+            </form>
+
+            <textarea id="terminal" class="form-control" cols="20" rows="6" readonly></textarea>
+
+            <script>
+              function discoverUserByEmails() {
+                var email = $('#discoverUserByEmails #email')[0].value
+
+                return logP($('#discoverUserByEmails #terminal')[0])(skygear.auth.discoverUserByEmails([email]))
+              }
+            </script>
+          </div>
+
+          <div id="discoverUserByUsernames">
+            <code>discoverUserByUsernames([username])</code>
+            <form class="form-inline" onsubmit="event.preventDefault(); discoverUserByUsernames()">
+              <div class="form-group mb-2">
+                <label for="username" class="sr-only">Username</label>
+                <input type="username" class="form-control" id="username" placeholder="Username" required>
+              </div>
+              <button type="submit" class="btn btn-primary mb-2">Discover</button>
+            </form>
+
+            <textarea id="terminal" class="form-control" cols="20" rows="6" readonly></textarea>
+
+            <script>
+              function discoverUserByUsernames() {
```

Comment:
It is great that we have a clear function name for each demo! 

Maybe it will look even better if we group them all in a single script block and put them at the end of the body. It would be easier for user to have a quick preview of all the api, and easier for them to find what they want. It would be great if we can have comments before the functions too!